docs/quickstart/launch.md

```diff
@@ -40,12 +35,13 @@ After a few moments you should see the lakeFS container ready to use:
 
 You're now ready to dive into lakeFS! 
 
-1. Login to lakeFS's web interface at [http://127.0.0.1:8000/](http://127.0.0.1:8000/) using these credentials:
+1. Open lakeFS's web interface at [http://127.0.0.1:8000/](http://127.0.0.1:8000/), enter your email address, and then click **Setup**.
 
-    * **Access Key ID**: `AKIA-EXAMPLE-KEY`
-    * **Secret Access Key**: `EXAMPLE-SECRET`
+2. Make a note of the Access Key ID and Secret Access Key and click on **Go To Login**. 
 
-2. You'll notice that there aren't any repositories created yet. Click the **Create Sample Repository** button. 
+3. Login with the credentials that you've just created. 
+
+4. You'll notice that there aren't any repositories created yet. Click the **Create Sample Repository** button. 
```

Comment:
My favourite Markdown tip: Number all your lists "1.", "1.", "1.", "1.", ...  Then diffs are easier to read when you add or remove an item from them 😛

Reply:
ah yes, sorry - you told me this one and I keep forgetting :)